packages/www/components/Dashboard/SessionsTable/index.tsx

```diff
@@ -96,7 +96,10 @@ const StreamSessionsTable = ({
   mt?: string | number;
 }) => {
   const { user, getStreamSessions } = useApi();
-  const tableProps = useTableState({ pageSize: 50 });
+  const tableProps = useTableState({
+    pageSize: 50,
```

Comment:
Our table page size is 20 per row. Should this default be set to 20?

Reply:
done